dependencies.gradle

```diff
@@ -5,8 +5,7 @@ ext {
   retroLambdaPatch = 'me.tatarka.retrolambda.projectlombok:lombok.ast:0.2.3.a2'
   compileSdkVersion = 25
   buildToolsVersion = '25.0.1'
-  minSdkVersionDebug = 21
-  minSdkVersion = 17
+  minSdkVersion = 16
```

Comment:
Good call!

I'll add it as a task to do here.

Looks like a small price to pay to be able to reach 4.5% (https://developer.android.com/about/dashboards/index.html - as of Dec 19) of the devices out there. :)